SwaprHQ/stackly-ui
# [STK-233][FIX] - Bug in UI order amount calcs

## Fixes: [STK-233](https://linear.app/swaprhq/issue/STK-233/bug-in-ui-order-amount-calcs)

# Description
* Update the logic on order creation flow for submitted dates
* Something is still happening, I think it's an issue happening after we submit the data on the transaction sign

# How to test the changes
![image](https://github.com/SwaprHQ/stackly-ui/assets/21271189/3bf42275-8bec-4f5f-ba50-48b631efe997)
![image](https://github.com/SwaprHQ/stackly-ui/assets/21271189/a09e09d1-3acb-4361-b147-c4e54aa93ea8)


1) Pull this branch
2) Run the project locally
3) Go to Stackly dApp
4) Connect your wallet
5) Create a stack, take note of the number of orders to be executed in the Create Order modal
6) Go to Your Stacks, then check the recently created Stack and it should have the same number of orders to be executed
